RecoLocalCalo/Castor/src/RecHitCorrector.cc

```diff
@@ -163,11 +163,14 @@ RecHitCorrector::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 	if (ok) {
 	    CastorRecHit *correctedhit = new CastorRecHit(rechit.id(),correctedenergy,time);
```

Comment:
why all this?
rec.emplace_back(rechit.id(),correctedenergy,time)
should suffice